tools/IceRpc.Slice.Tools/SliceCCSharpTask.cs

```diff
@@ -145,4 +158,48 @@ protected override void LogEventsFromTextOutput(string singleLine, MessageImport
 
     /// <inheritdoc/>
     protected override void LogToolCommand(string message) => Log.LogMessage(MessageImportance.Normal, message);
+
+    /// <inheritdoc/>
+    protected override int ExecuteTool(string pathToTool, string responseFileCommands, string commandLineCommands)
+    {
+        var startInfo = new ProcessStartInfo
+        {
+            FileName = pathToTool,
+            Arguments = commandLineCommands,
+            RedirectStandardOutput = true,
+            UseShellExecute = false,
+            CreateNoWindow = true,
+            WorkingDirectory = GetWorkingDirectory()
+        };
+
+        using (var process = Process.Start(startInfo))
+        {
+            if (process == null)
+            {
+                Log.LogError("Failed to start the Slice compiler process.");
+                return -1;
+            }
+
+            string output = process.StandardOutput.ReadToEnd();
+            process.WaitForExit();
+
+            if (Verbose == true)
+            {
+                try
+                {
+                    var jsonDoc = System.Text.Json.JsonDocument.Parse(output);
+                    OutputHash = jsonDoc.RootElement.GetProperty("hash").GetString();
+                    UpdatedFiles = jsonDoc.RootElement.GetProperty("updated_files").GetBoolean();
+                }
+                catch (Exception)
+                {
+                    // Don't fail the build if we can't parse the output
```

Comment:
Why not?

Something is very wrong if slicec-cs emits unexpected output to stdout.